ttnn/ttnn/operations/matmul.py

```diff
@@ -8,6 +8,16 @@
 import ttnn
 
 MatmulProgramConfig = ttnn.experimental.operations.primary.MatmulProgramConfig
+MatmulMultiCoreReuseProgramConfig = ttnn.experimental.operations.primary.MatmulMultiCoreReuseProgramConfig
+MatmulMultiCoreReuseMultiCastProgramConfig = (
+    ttnn.experimental.operations.primary.MatmulMultiCoreReuseMultiCastProgramConfig
+)
+MatmulMultiCoreReuseMultiCast1DProgramConfig = (
+    ttnn.experimental.operations.primary.MatmulMultiCoreReuseMultiCast1DProgramConfig
+)
+MatmulMultiCoreReuseMultiCastDRAMShardedProgramConfig = (
+    ttnn.experimental.operations.primary.MatmulMultiCoreReuseMultiCastDRAMShardedProgramConfig
+)
```

Comment:
Eventually we should move the implementation and pybinding of these structs into ttnn

Reply:
Yes. That'll be a future PR. First I need to move them from other places as well. Mainly models. Then I'll move the pybinding into ttnn.